examples/pip_install_incremental/requirements_lock.txt

```diff
@@ -0,0 +1,11 @@
+# Transitive dependency set from :requirements.txt
```

Comment:
nit: maybe give what command you used to generate this

Reply:
I used pip-compile which is not a tool we are supporting afaict, but I will add that.

Reply:
What does it mean to "support" a tool? Does it matter how a pinned requirements file is generated? I personally have only ever used pip-compile

Reply:
I meant we don't have any rules/helpers/tooling around it in the repo. I think its fine to suggest it, though it won't work for everyone, which is why I mentioned using a virtualenv and pip freeze in the README.